CSharpSourceCode/Abilities/Scripts/SummoningScript.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace TOW_Core.Abilities.Scripts
+{
+    public class SummoningScript : AbilityScript
```

Comment:
Beginning of something? Remove for now?

Reply:
its how we did with the others too, even if empty. The factory needs it to assign behaviour, even if it does nothing compared to the basic AbilityScript. I guess we could just return AbilityScript in the factory, but i like it like this :)

Reply:
Fair enough. Its easier to reason about spell types in behaviors this way as well.